settings_page.php

```diff
@@ -0,0 +1,233 @@
+<?php
+
+// A class to create and manage the admin's B2C settings page
+class settings_page
+{
+    /**
+     * Holds the values to be used in the fields callbacks
+     */
+    private $options;
+
+    /**
+     * Start up
+     */
+    public function __construct()
+    {
+        add_action( 'admin_menu', array( $this, 'add_plugin_page' ) );
+        add_action( 'admin_init', array( $this, 'page_init' ) );
+    }
+
+    /**
+     * Adds a B2C options page under "Settings"
+     */
+    public function add_plugin_page()
```

Comment:
>add_plugin_page [](start = 20, length = 15)

Try to keep the same naming convention. Look at EndpointHandler which is using camelCase for method name and without "_"